README.md

```diff
@@ -1,4 +1,4 @@
-# ModestaCSS
+# ModestaCSS 2.0
```

Comment:
Instead of making this say 2.0, make a new title that showcase where to find all legacy versions (can be found on ModestaCSS docs